src/index.md

```diff
@@ -4,14 +4,78 @@
+
+Once you are deemed ready, a Reviewer in the project can start a vote by sending an email proposing that you should become a Reviewer. The email should follow the template found in the [OpenJDK Project description](https://openjdk.java.net/projects/).
+
+### Non-trivial/Significant changes
+
+One key definition when advancing through the OpenJDK roles is the significant change. What exactly does it take for a change to be significant?
+
+Instead of describing the significant change (because that's quite difficult to define) provided here is a few examples of changes that wouldn't be considered significant or for other reasons wouldn't count as significant contributions.
+
+* Purely aesthetic changes like renaming or fixing indentation
+* Repeated follow-up bugfixes from earlier changes
+* Larger changes where only a non-significant portion of the work was done by the Contributor under vote
+* Trivial backports of someone else's changes
+
+# Contributing to an OpenJDK Project
+
+Contributing to OpenJDK can take many forms. Writing code and providing patches is just one of them. A big part of developing a feature or a bugfix is testing and code review. Anything you can do to help out in these areas will be recognized as a contribution. Join the [mailing lists](#mailing-lists) to engage in design discussions and reviews, and download the latest EA builds or project repositories to try out new features and give feedback.
+
+Anything you can do to spread the word about Java, new features, and your experiences using the JDK will be helpful for the community and to the OpenJDK developers. Trying out a new feature and reporting your experiences is also a contribution. Whether you find that the new feature improves your application, or if you find some area that needs to be improved, your feedback is valuable to the developers of that feature.
+
+If you have a success story where Java solved your problem, or if you successfully upgraded to a more recent version of the JDK and noticed some improvements, spreading this story through a blog, news article, or some other channel is also a contribution.
+
+If you're in a position to choose what programming language to use in a project, in a tutorial, or in a class, you have the power to enlarge the Java community in a very direct way, and your colleagues or students will get an opportunity to learn one of the most used programming languages in the world.
```

Comment:
This is a good section, talking about contributions in general, not just code like everybody thinks it is.

One thing I'd suggest adding is finding and diagnosing bugs. If you see some misbehavior, or if you see somebody mention some misbehavior on some internet forum, try to track it down. Good bug reports with reproducible test cases are extremely valuable and make excellent contributions.

Reply:
Added your suggestion.